nss-day-cohort-39/hello-world-where-do-we-want-to-go
# Coding standards updated and placement of index.html

Changes that were made:

-Put france page into france.html and made travel the index.html
-Updated the nav bar links to the corresponding pages after updating index.html
-Changed the styling of dialog boxes in index.html
-Added footer.css, background.css, header.css, and nav.css and put the corresponding css into these files
-Updated wording of how we targeted items 

Steps to test:

1. Make sure that the nav bar navigates to the correct pages 
2. Make sure that travel page loads first
3. Look at the css files to ensure that the corresponding css is in them
4. Make sure that the new wording in for the coding standards makes sense
5. Make sure that the picture styling on the index.html looks nice and aggreeable

THANK YOU TEAM, you guys are awesome!! :)
